taskcall/README.md

```diff
@@ -0,0 +1,113 @@
+
+### Create Webhook in Datadog
+
+1. Go to [**Integrations** > **Integrations**][7].
+2. Find **Webhooks** and click on it.
+3. Click on the **New Webhook** button.
+4. Give it a name and paste the **Integration Url** you copied over from TaskCall.
+5. Copy the following [JSON payload][2] and paste it in the Payload section.
+```json
+{
+     "body": "$EVENT_MSG",
+     "last_updated": "$LAST_UPDATED",
+     "event_type": "$EVENT_TYPE",
+     "title": "$EVENT_TITLE",
+     "date": "$DATE",
+     "org": {
+         "id": "$ORG_ID",
+         "name": "$ORG_NAME"
+     },
+     "id": "$ID",
+     "aggreg_key": "$AGGREG_KEY",
+     "alert": {
+         "cycle_key": "$ALERT_CYCLE_KEY",
+         "id": "$ALERT_ID",
+         "metric": "$ALERT_METRIC",
+         "scope": "$ALERT_SCOPE",
+         "status": "$ALERT_STATUS",
+         "title": "$ALERT_TITLE",
+         "transition": "$ALERT_TRANSITION",
+         "type": "$ALERT_TYPE",
+         "query": "$ALERT_QUERY"
+     },
+     "user": "$USER",
+     "username": "$USERNAME",
+     "priority": "$PRIORITY",
+     "text_msg": "$TEXT_ONLY_MSG",
+     "snapshot": "$SNAPSHOT",
+     "link": "$LINK",
+     "hostname": "$HOSTNAME",
+     "incident_uuid": "$INCIDENT_UUID",
+     "incident_public_id": "$INCIDENT_PUBLIC_ID",
+     "incident_title": "$INCIDENT_TITLE",
+     "incident_url": "$INCIDENT_URL",
+     "incident_msg": "$INCIDENT_MSG",
+     "incident_severity": "$INCIDENT_SEVERITY",
+     "security_rule_id": "$SECURITY_RULE_ID",
+     "security_rule_name": "$SECURITY_RULE_NAME",
+     "security_signal_severity": "$SECURITY_SIGNAL_SEVERITY",
+     "security_signal_title": "$SECURITY_SIGNAL_TITLE",
+     "security_signal_msg": "$SECURITY_SIGNAL_MSG",
+     "security_rule_query": "$SECURITY_RULE_QUERY",
+     "security_rule_type": "$SECURITY_RULE_TYPE",
+     "tags": "$TAGS"
+}
+```
+6. Once the details have been entered, click Save.
+
+For more information, see the [TaskCall Datadog Integration Guide][3].
+
+## Uninstallation
+
+- In TaskCall, delete the integration from Services > Integrations.
+- Navigate to the [TaskCall integration tile][6] in Datadog and uninstall it. You must also delete the webhook you created to send notifications to TaskCall.
+- Once this integration has been uninstalled, any previous authorizations are revoked.
+
+
+## Support
+
+[Contact TaskCall support][4] for any questions regarding the integration or the platform.
+
+
+[1]: https://app.us.taskcallapp.com/register
+[2]: https://docs.taskcallapp.com/integrations/v1/datadog-integration-guide#in-datadog
+[3]: https://docs.taskcallapp.com/integrations/v1/datadog-integration-guide
+[4]: https://www.taskcallapp.com/contact-us
+[5]: https://raw.githubusercontent.com/DataDog/integrations-extras/master/taskcall/images/DatadogTaskCallAuthorization.png
+[6]: https://app.datadoghq.com/integrations/taskcall
+[7]: https://app.datadoghq.com/integrations
```

Comment:
Looks like this instruction was accidentally removed (I also added the link to the api keys page).

```suggestion
- Additionally, ensure that all API keys associated with this integration have been disabled by searching for the integration name on the [API Keys page][8].


## Support

[Contact TaskCall support][4] for any questions regarding the integration or the platform.


[1]: https://app.us.taskcallapp.com/register
[2]: https://docs.taskcallapp.com/integrations/v1/datadog-integration-guide#in-datadog
[3]: https://docs.taskcallapp.com/integrations/v1/datadog-integration-guide
[4]: https://www.taskcallapp.com/contact-us
[5]: https://raw.githubusercontent.com/DataDog/integrations-extras/master/taskcall/images/DatadogTaskCallAuthorization.png
[6]: https://app.datadoghq.com/integrations/taskcall
[7]: https://app.datadoghq.com/integrations
[8]: https://app.datadoghq.com/organization-settings/api-keys?filter=TaskCall
```

Reply:
We were asked to remove this line by @anawishnoff since we are not using any API keys.

Reply:
Oh, got it. Sorry, I didn't realize you weren't using that scope. Thanks for clarifying :) 

I'll go approve!